src/publish/mergePublishBranch.ts

```diff
@@ -21,3 +30,15 @@ export function mergePublishBranch(publishBranch: string, branch: string, messag
   }
   return result!;
 }
+
+/**
+ * Calls a specified hook for each package being bumped
+ */
```

Comment:
Just realized this comment is probably an incorrect copy/paste
```suggestion
```

Reply:
uuuh no that's correct ?